src/traits.rs

```diff
@@ -62,7 +62,7 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
 
         for (i, method) in ::rustc::traits::get_vtable_methods(self.tcx, trait_ref).enumerate() {
             if let Some((def_id, substs)) = method {
-                let instance = ::eval_context::resolve(self.tcx, def_id, substs);
+                let instance = resolve(self.tcx, def_id, substs);
```

Comment:
In general, makes it easier to see that it comes from *somewhere else*. At least for functions.